app/uk/gov/hmrc/transitmovementsrouter/connectors/AuditingConnector.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.connectors
+
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import com.google.inject.ImplementedBy
+import com.google.inject.Inject
+import com.kenshoo.play.metrics.Metrics
+import io.lemonlabs.uri.UrlPath
+import play.api.Logging
+import play.api.http.HeaderNames
+import play.api.http.Status.ACCEPTED
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.http.StringContextOps
+import uk.gov.hmrc.http.UpstreamErrorResponse
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.transitmovementsrouter.config.AppConfig
+import uk.gov.hmrc.transitmovementsrouter.metrics.HasMetrics
+import uk.gov.hmrc.transitmovementsrouter.metrics.MetricsKeys
+import uk.gov.hmrc.transitmovementsrouter.models.AuditType
+import uk.gov.hmrc.transitmovementsrouter.models.EoriNumber
+import uk.gov.hmrc.transitmovementsrouter.models.MessageId
+import uk.gov.hmrc.transitmovementsrouter.models.MessageType
+import uk.gov.hmrc.transitmovementsrouter.models.MovementId
+import uk.gov.hmrc.transitmovementsrouter.models.MovementType
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.control.NonFatal
+
+@ImplementedBy(classOf[AuditingConnectorImpl])
+trait AuditingConnector {
+
+  def post(
```

Comment:
@drnaylor-oc / @prashanthmamidi are we not going to include statusType audit event here? Con't find them but it is requited I think?

Reply:
@sonisn1988 yes it's needed, will add this.  I think when I copied this file from CTC traders  service this audit status Type changes are not available at that time.